index.html

```diff
@@ -157,87 +157,87 @@ <h2>Stages</h2>
 
-<h2>Conditional Advancement</h2>
+<h2>Conditional advancement</h2>
 
-<p>A delegate may also request additional time to consider the proposal, if a topic they had not considered comes up during discussion. In this case, the delegate should give the champion some actionable request for how to facilitate the analysis (e.g., the champion could walk through the proposal with the delegate offline). In practice, this work should be done during the plenary, or before the next meeting. A delegate may also request additional time to consider the proposal, if it was added to the agenda after the deadline for proposal advancement.
+<p>A delegate may request additional time to consider the proposal if a topic they had not considered comes up during discussion. In this case, the delegate should give the champion an actionable request for how to facilitate the analysis (e.g., the champion could walk through the proposal with the delegate offline). In practice, this work should be done during the plenary, or before the next meeting. A delegate may also request additional time to consider the proposal, if it was added to the agenda after the deadline for proposal advancement.
 
 <p>The committee may resolve to <em>conditionally advance</em> a proposal to address a particular well-understood condition offline, e.g., making a particular small specification change concrete, among a group of interested people who have an idea of the solution. Conditional advancement is time-limited, giving the person raising the concern time to discuss with the champions and authors about their concerns. If a proposal has a conditional advancement, an issue must be opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.
 
-<h2>Withdrawing Proposals, Reverting to Earlier Stages, and Adopting Proposals</h2>
+<h2>Withdrawing proposals, reverting to earlier stages, and adopting proposals</h2>
 
-<p>At any point in the process, a proposal champion may propose that a proposal be downgraded to an earlier stage or withdrawn. Consensus of the committee is necessary for these transitions. The proposal to make this change must be accompanied by a reason why it is appropriate, e.g., a significant issue that may have not been considered, or identified, before.
+<p>At any point in the process, a proposal champion may propose downgrading a proposal to an earlier stage, or withdrawing it altogether. Consensus of the committee is necessary for these transitions. The proposal to make this change must include a rationale explaining why it is appropriate, e.g., a significant issue that may have not been considered, or identified, before.
```

Comment:
It would be great to use a more obvious antonym of "advance".
```suggestion
<p>At any point in the process, a proposal champion may propose regressing a proposal to an earlier stage, or withdrawing it altogether. Consensus of the committee is necessary for these transitions. The proposal to make this change must include a rationale explaining why it is appropriate, e.g., a significant issue that may have not been considered, or identified, before.
```

Reply:
I much prefer `regress` and that's the word I use when speaking so +1 from me

Reply:
I like promote/demote, but since we already use "advance", "regress" is fine.

Reply:
also updated the other instance of `downgrade` in the doc, commit to be pushed momentarily